src/Sulu/Bundle/SecurityBundle/Resources/public/js/components/roles/edit/details/main.js

```diff
@@ -18,7 +18,8 @@ define([
         permissionData,
         matrixContainerSelector = '#matrix-container',
         matrixSelector = '#matrix',
-        formSelector = '#role-form';
+        formSelector = '#role-form',
+        noneSelected = false;
```

Comment:
Basically the same reasons as in https://github.com/massiveart/husky/pull/717#discussion_r87024937